internal/cmd/controller/root.go

```diff
@@ -54,6 +54,12 @@ type LeaderElectionOptions struct {
 	RetryPeriod *time.Duration
 }
 
+type ControllerReconcilerWorkers struct {
+	Gitrepo          int
```

Comment:
I just noticed it's GitRepo everywhere else

```suggestion
	GitRepo          int
```

Reply:
changed !